File: 001/001.js
var sum = 0;

(function(cb) {
  for (var i = 1; i < 1000; i++) {
    if (i % 3 === 0) {
      sum += i;
    } else if (i % 5 === 0) {
      sum += i;
    }
  }
  cb();
})(function() {
  console.log(sum);
});

File: 001/001.c
#include <stdio.h>

int sum = 0;

int main(void)
{
  for (int i = 1; i < 1000; i++) {
    if (i % 3 == 0) {
      sum += i;
    } else if (i % 5 == 0) {
      sum += i;
    }
  }

  printf("%d\n", sum);

  return 0;
}
